src/main/java/org/broadinstitute/hellbender/Main.java

```diff
@@ -147,9 +155,16 @@ public Object instanceMain(final String[] args) {
      * Note: this is the only method that is allowed to call System.exit (because gatk tools may be run from test harness etc)
      */
     protected final void mainEntry(final String[] args) {
-        final CommandLineProgram program = extractCommandLineProgram(args, getPackageList(), getClassList(), getCommandLineName());
+
+        // First we package our args together a little better:
+        ArrayList<String> argArrayList = new ArrayList<>( Arrays.asList(args) );
+
+        // Now we setup our configurations:
+        ConfigUtils.initializeConfigurationsFromCommandLineArgs(argArrayList, gatkConfigFileOption);
```

Comment:
For the integration tests, we may not need to change this.  This method initializes configuration files based on the command-line option to pass a configuration file.  If you don't call this method the configuration values will be set to default values in the GATKConfig interface itself (or to a default file, but I haven't set that up).